src/digraph13/transitivity.jl

```diff
@@ -157,48 +157,12 @@ julia> collect(edges(transitivereduction(barbell)))
 function transitivereducion end
 @traitfn function transitivereduction(g::::IsDirected; selflooped::Bool=false)
     scc = strongly_connected_components(g)
-    cg = condensation(g, scc)
-
-    reachable = Vector{Bool}(undef, nv(cg))
-    visited = Vector{Bool}(undef, nv(cg))
-    stack = Vector{eltype(cg)}(undef, nv(cg))
+    cg = condensation(g, reverse!(scc))
```

Comment:
this is unintuitive. I would separate the `reverse!` into a separate call and then just use `scc` from there on out.
```
reverse!(scc)
cg = condensation(g, scc)
```

or, even better, on like 159,
```
scc = reverse(strongly_connected_components(g)
```